package lotte.newdevps.exception.http;

import lotte.newdevps.exception.NewdevpsException;

public class InternalException extends NewdevpsException {
}
